### PR TITLE
Work on API Integration

### PR DESCRIPTION
**Pull request by jleach**
Initial work on API integration for signing services. Don't spend too much time on the React components, I'm going to refactor App into smaller components. This was put together hastily so that I could validate end-to-end (Web->API-Agent->API->Web) functionality. Feedback is still welcome :)

### CODE REVIEW

File: src/actionCreators/index.js [hunk 1]
```diff
@@ -19,31 +19,71 @@
...
+  jobCreated,
+  jobCreationFailed,
+  jobProcessing,
+  jobCompleted,
+} from '../actions';
+import { API } from '../constants';
 
-export const createSigningJob = files => dispatch => {
-  console.log('** HERE **', files);
+axios.defaults.baseURL = API.BASE_URL();
```
Comment (Kyubinhan) [inline src/actionCreators/index.js]: mutating global variable like this doens't seem to be right to me. maybe you can create an instance?
const instance = axios.create({
  baseURL: API_BASE_URL,
});

File: src/actionCreators/index.js [hunk 2]
```diff
@@ -19,31 +19,71 @@
...
+
+        return;
+      }
+
+      if (res.status === 200 && res.data.status === 'Completed') {
+        dispatch(jobCompleted(res.data));
+      }
+    })
+    .catch(err => {
+      console.log(`error = ${err.message}`);
```
Comment (Kyubinhan) [inline src/actionCreators/index.js:90]: dispatching an action would be great here

File: src/actionCreators/index.js (hunk 1 above)
Comment (jleach) [reply src/actionCreators/index.js]: Fixed.

File: src/actionCreators/index.js (hunk 2 above)
Comment (jleach) [reply src/actionCreators/index.js:90]: Done.

File: src/actionCreators/index.js [hunk 3]
```diff
@@ -19,31 +19,75 @@
...
+  jobStatusCheckFailed,
+} from '../actions';
+import { API } from '../constants';
 
-export const createSigningJob = files => dispatch => {
-  console.log('** HERE **', files);
+const axi = axios.create({
+  baseURL: API.BASE_URL(),
+});
+
```
Comment (Kyubinhan) [inline src/actionCreators/index.js:35]: FYI If you prefer using the name axios as a variable you can do:
import baseAxios from 'axios'; // or whatever you wanna name
const axios = baseAxios.create({
  baseURL: API.BASE_URL(),
});

Comment (jleach) [reply src/actionCreators/index.js:35]: I'm OK with this. Thx.

**Review by Kyubinhan — APPROVED** [review]

**Pull request merged**